plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -199,6 +200,7 @@ export class PgInstrumentation extends InstrumentationBase {
             .then((result: unknown) => {
               // Return a pass-along promise which ends the span and then goes to user's orig resolvers
               return new Promise(resolve => {
```

Comment:
I understand that nested `new Promise` was here before but it really is not needed.
You can just do the sync calls inside the then callback function and it behaves exactly the same.

Reply:
thanks @Rauno56, I agree and also noticed it when writing this PR. However, and if that is alright with you, I'd prefer to leave this change to a different PR?